---
title: "Projects"
permalink: /projects/
---

---

# Metabolic dysregulation in drug-resistant strains of _M. tuberculosis_ (2022-Current)
Drug-resistant strains of Mycobacterium tuberculosis are a major global health problem. Resistance to the front-line anti-tubercular isoniazid is often associated with mutations in the katG encoded bifunctional catalase-peroxidase. We hypothesised that perturbed KatG activity would generate in collateral vulnerabilities in INH-resistant katG mutants, providing new pathways to combat isoniazid resistance. Here using whole genome CRISPRi screens, transcriptomics, and metabolomics we have generated a genome-wide map of cellular vulnerabilities in a M. tuberculosis katG mutants. Specifically, we discovered that metabolic and transcriptional remodelling is used to compensate for the loss of KatG, generating vulnerabilities in ribosome biogenesis, and nucleotide and amino acid metabolism. These vulnerabilities were more sensitive to inhibition in an isoniazid-resistant katG mutant under in vitro and host-relevant conditions and translated to clinical isolates. These findings provide a framework for developing novel strategies to combat antimicrobial resistance in M. tuberculosis and other bacterial pathogens.

A manuscript is under preparation for submission

---

# Our Land and Water (2019-2021)

Land and water are the most valuable sources. In New Zealand, agriculture is dominanted by large pasture farms with large quantities of ruminant animals. Although everything looks natural and green, pasture farms and the ruminants over time,can result in undesirable environmental outcomes, especially in a long term. 

Ruminant animals such as cows produce large quantities of CH<sub>4</sub> gas, a major greenhouse gas second to CO<sub>2</sub>, contributing to climate changes. Apart from emitting greenhouse gas, ruminants also introduce excessive nitrogen to the local ground and the nearby water sources, potentially damaging the soil ecosystem and causing water eutrophication. 

The goal of the OLW project was to improve the land sustainabilities base on a comprehensive understanding of the ecological processes and the associated underlying mechanisms.

My involvement in OLW projects was to understand microbiomes across farm niches and their interactions, based on amplicon and omics data. In addition, by introducing a distrubance (urea) to soils, we would be able to test if and how the neighbourhood microbiomes respond to disturbances in soils. 

The study had been publishsed in Fronteris in Microbiology titled "Soil Nitrogen Treatment Alters Microbiome Networks Across Farm Niche" [Paper link](https://www.frontiersin.org/articles/10.3389/fmicb.2021.786156/full)
![Graphical abstract](../images/olw_ab.jpg)

---

# Alternative Hydrogen Sinks in the Rumen Environment (2018-2019)
Livestock farming is a vital part of agriculture that has significant environmental impacts. According to  [Hannah Ritchie and Max Roser](https://ourworldindata.org/land-use),  total of 77% of agricultural land is allocated to livestock, accounting for 37% of the global protein supply.

Enteric methane (CH<sub>4</sub>) production is a waste of livestock food intake energy, contributing to global warming. Approximately 3.3 gigatons of enteric CH4 is released every year globally, leading to 2 to 12% of food intake loss for the ruminant livestock sector. Therefore, mitigating enteric CH4 remains an urgent need globally. The methanogenic archaea responsible for these emissions use molecular hydrogen (H<sub>2</sub>), produced during bacterial and eukaryotic carbohydrate fermentation, as their primary energy source.

In this work, we used comparative genomic, metatranscriptomic, and co-culture-based approaches to gain a system-wide understanding of the organisms and pathways responsible for ruminal H2 metabolism. By screening differentially abundant rumen micro-organism  between low and high CH4 yield sheep, potential pathways and the associated micro-organisms may help to benifit further development of CH<sub>4</sub> mitigating strategies.

My role in this project was to develop and automate a pipeline to identify target genes that were either differentially abundant or differentially expressed using metagenome and metatranscriptomic data.

The study had been published in The ISME Journal titled Diverse hydrogen production and consumption pathways influence methane production in ruminants
 [Paper link](https://www.nature.com/articles/s41396-019-0464-2)

## Investigating the Mechanism of Fumarate Addition in Reducing Enteric Methane Emissions (2019-2021)
A follow-up study was performed to test the proposed hydrogen competion between potential [H] sinks in the rumen environment. 
Manuscript in preparation


<!-- {% include base_path %}


{% for post in site.projects %}
  {% include archive-single.html %}
{% endfor %}
-->

